CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.6.0 (November 20, 2020)
+
+- Update dependencies to support TypeScript 4, ESLint 7 and Prettier 2
+- Enable `@typescript-eslint/no-unused-before-define` rule in TypeScript files
+- Enable `@typescript-eslint/prefer-optional-chain` rule
+- Enable `unicorn/no-null` rule
+- Enable `unicorn/no-reduce` rule
```

Comment:
I do believe we use reduce in areas of statement generation, but I agree it's not readable.

Reply:
This is just a warning. Sometimes it does make sense to use `reduce` and you can just disable the rule.